sagemaker_batch_transform/pytorch_flores_batch_transform/flores_inputs/flores101-small6-devtest.json

```diff
@@ -0,0 +1,200 @@
+{"uid": "83-est-eng-devtest", "sourceLanguage": "est", "targetLanguage": "eng", "sourceText": "Ninth Wardis, mida tabas orkaan Katrina ajal tervelt 20 jala kõrgune üleujutus, on praegu vöökõrgune vesi, kuna lähedal asuv tamm hakkas üle ajama.", "targetText": "The Ninth Ward, which saw flooding as high as 20 feet during Hurricane Katrina, is currently in waist-high water as the nearby levee was overtopped."}
```

Comment:
What is the source of this "flores" dataset?

Reply:
@jkroll-aws Flores dataset is open source can be found [here](http://www.statmt.org/wmt21/large-scale-multilingual-translation-task.html?fbclid=IwAR20x8ZIe9DeVYmBW7y-H9nLaTAoKqIfd2_KFzw99ru-JZ4NnkylRBTsfJA,), that needs a data transformation, that we skipped that part and just uploaded the transformer json files to simplify the process for users.

Reply:
Thanks for the link. Just because a dataset can be downloaded online does not mean that it can be used for commercial use. This dataset has not been reviewed by legal, and does not list a standard license that I can find. I do see that they say `This data is provided for the WMT21 Shared Task on Large-Scale Multilingual Machine Translation but you may use it for any other purpose` - that's a good sign but not sufficient on its own. If you cannot use an already-approved dataset, then we will need to start a legal review of Flores. Please note that this puts your notebook launch at risk in case the legal review is not completed on time, or the dataset is ultimately denied.

Reply:
I found the original source: https://github.com/facebookresearch/flores
This is under a CC-BY-SA-4.0 (sharealike) license, so you cannot distribute a modified version of the dataset, or mix with other data. This requires legal review to use.

Reply:
Done, thanks